### DISCUSSION TITLE
Upload Package To Microsoft Store Failed since updated to project reunion 8.5

### DISCUSSION THREAD

**Discussion by giraudremy** [microsoft/WindowsAppSDK#1695] [Dev team discussions]
Hi,

I have update my projet to 8.2 => 8.5, I have generate my package to update my store app, but when I upload the package to the Microsoft Store I have this error :

Votre package Microsoft.ProjectReunion.0.8 spécifie la version 8005.278.2204.0, alors que 8000.167.1906.0 est la version minimale disponible.

I don't know what I need to change to solve this issue.

Any Idea ?

Thank

**Comment by giraudremy** [comment]
It's ok with version 0.8.4. It look like similar with this discussion 
[https://github.com/microsoft/WindowsAppSDK/discussions/1726](url)

**Comment by andrewleader** [comment]
Thanks @giraudremy for reporting this, we're looking into it!

**Reply by andrewleader** [reply]
@giraudremy, we're tracking this issue with issue #1749. Please subscribe to #1749 to get updates on fixing being able to publish apps using 0.8.5 to the Store!

**Reply by andrewleader** [reply]
0.8.5 is now available in the Store and you should be able to release your app now @giraudremy!

**Reply by giraudremy** [reply]
Thank, it's good now 👍
